cmp/grid/GridModel.js

```diff
@@ -130,6 +130,8 @@ export class GridModel {
     @observable showSummary = false;
     /** @member {string} */
     @observable emptyText;
+    /** @member {string} */
+    @observable treeGridStyleMode; // Not sure this can't be immutable, but for our example I need observable
```

Comment:
Maybe safer to leave observable for as yet unimagined use cases?

Reply:
no, I don't think so.  There are many immutable properties on the GridModel.  Also, can we call it 'treeStyle'?

Reply:
Sounds good. And yeah that name is bulky, although last I spoke to Anselm he said "let us leave some room to grow by using a config with “mode” in the name". Maybe we go with treeStyleMode as grid is really the obv throw away